listenbrainz/spark/handlers.py

```diff
@@ -48,6 +50,7 @@ def notify_user_stats_update(stat_type):
             from_name='ListenBrainz',
             from_addr='noreply@'+current_app.config['MAIL_FROM_DOMAIN']
         )
+        metrics.set("listenbrainz-spark-user-stats", 1)
```

Comment:
This function is unused. I have cleaned it up in the couchdb PR, so should remove this metrics.set.